frontend/src/app/pages/exercises/exercise/shared/core/drag-element.service.ts

```diff
@@ -280,21 +282,33 @@ export class DragElementService {
                     // This ratio has been determined by trial and error
                     const height = SimulatedRegion.image.height / 23.5;
                     const width = height * SimulatedRegion.image.aspectRatio;
-                    const simulatedRegion = SimulatedRegion.create(
-                        {
-                            x: position.x - width / 2,
-                            y: position.y + height / 2,
-                        },
-                        {
-                            height,
-                            width,
-                        },
-                        'Einsatzabschnitt ???'
+                    const simulatedRegion = cloneDeepMutable(
+                        SimulatedRegion.create(
+                            {
+                                x: position.x - width / 2,
+                                y: position.y + height / 2,
+                            },
+                            {
+                                height,
+                                width,
+                            },
+                            'Einsatzabschnitt ???',
+                            ''
+                        )
                     );
+                    const transferPoint = TransferPoint.create(
+                        SimulatedRegionPosition.create(simulatedRegion.id),
+                        {},
+                        {},
+                        '',
```

Comment:
Because it is immutable and defined, it has to be set somewhere. I felt like it was cleaner to do it this way instead of always setting it in the constructor.

Reply:
Maybe I would create the TransPoint at some point during construction of the SimReg